src/lib/udev/google_nvme_id

```diff
@@ -58,7 +58,7 @@ function err() {
 #######################################
 function get_namespace_device_name() {
   local nvme_json
-  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | xxd -p -seek 384 | xxd -p -r)"
+  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | dd bs=1 skip=384 2>/dev/null | od -x)"
```

Comment:
IMU, should remove ` | od -x`, as it converts to hexadecimal.

```
[root@hhei-fcos ~]# grep dd /usr/lib/udev/google_nvme_id
  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | dd bs=1 skip=384 2>/dev/null | od -x)"
[root@hhei-fcos ~]# /usr/lib/udev/google_nvme_id -d /dev/nvme0n1
[2023-07-27T13:37:34+0000]: Empty name

[root@hhei-fcos ~]# grep dd /usr/lib/udev/google_nvme_id
  nvme_json="$("$nvme_cli_bin" id-ns -b "$1" | dd bs=1 skip=384 2>/dev/null)"
[root@hhei-fcos ~]# /usr/lib/udev/google_nvme_id -d /dev/nvme0n1
/usr/lib/udev/google_nvme_id: line 61: warning: command substitution: ignored null byte in input
ID_SERIAL_SHORT=persistent-disk-0
ID_SERIAL=Google_PersistentDisk_persistent-disk-0
```

Reply:
fixed